### DISCUSSION TITLE
21 New event ideas

### DISCUSSION THREAD

**Discussion by Ha1fShade** [Gamer025/RainworldCE#1] [Ideas]
Here are 21 new event ideas, each with their own description and complementary names.
Enjoy!

- Auto balance - Sets a random karma value
- Bonk! - Stuns the player
- Shaded Citadel 2.0 - Covers the region in darkness
- /tp - Teleports the player to a random room in the region
- The rot - Changes all enemies in the room to ddls
- Spaaaaace! - Turns off gravity
- The eel - Turns an enemy in the room to a leviathan. If none, spawns one on the player
- Starvation - Drains the player’s food to 0 (maybe put them in starvation mode)
- Thanksgiving - Sets the player’s food to full, but the movement speed is a little slower
- Weather forecast - Sets the rain timer to a random amount (within reason)
- Acid gag reflex - drops the item in the player’s stomach out of it
- Hunter spawns - Changes the creature spawns to hunter’s for a cycle (I don’t know if you can do this mid cycle)
- Fling - Flings the player in a random x and y direction (within reason) in 5 seconds (or instantly)
- I’m feeling fancy - Gives the player a hat
- Invulnerability - Makes the player invincible for a short time
- Lag switch - Simulate lag for 15 seconds
- Sonic speed! - Doubles the players speed
- The pack - Spawns a pack of orange lizards in the region (maybe near the player)
- Inverted Controls - left becomes right, right becomes left. Up becomes down, down becomes up
- Porl - Spawns a randomly colored pearl with text on it (AI Generated???)
- I’m feeling lucky - Does a random event without telling the player what it is

**Comment by Gamer025** [comment]
Some of these definitely sound very interesting and also doable
I will check how feasible these are and hopefully will be able release and update including at least some of them in maybe the following weeks.

**Reply by Ha1fShade** [reply]
Glad to add to the chaos :)

**Comment by Gamer025** [comment]
Finally got around to this and im happy to announce most events made it into 1.1 in some form:

 * Auto balance - Added as 'Karma Level' event and less aggressive
 * Bonk! - Stuns the player - NOT added because it seemed to punishing with no way for the player to react, it either just slows the player down without consequences or leads to unavoidable deaths (climbing, being chased, etc.)
 * Shaded Citadel 2.0 - Added as 'Darkness / Shaded Region' event
 * /tp - Added
 * The rot - Added, spawns a DLL if no creature in room
 * Spaaaaace! - NOT added, actually wanted to add this as event when I created the mod but just leads to lots of issue, like the player going out of bounds and never returning if the room has no ceiling/walls/wasn't designed for no grav
 * The eel - Turns an enemy in the room to a leviathan. If none, spawns one on the player - NOT added, just didn't sound all that interesting. The leviathans don't do much on land and spawning one on top of the player is kinda unfair, also the 'DNA Mutation' event already has a chance to convert enemies into leviathans
 * Starvation - Added as 'Unending Hunger' event, but in a more interesting way
 * Thanksgiving - Added
 * Weather forecast - Added
 * Acid gag reflex - Added as 'Stomach ache' event
 * Hunter spawns - NOT added, don't think this is really feasible without lots of effort
 * Fling - Added as 'Hiccup' event
 * I’m feeling fancy - NOT added, would require way too much effort, is literally a separate full fledged mod (called HatWorld in case you are interested)
 * Invulnerability - Added as 'Second Chance' event in a more interesting way
 * Lag switch - NOT added, just sounds annoying without really doing anything cool, I know some people already are kinda annoyed by the "Super Slug" event
 * Sonic speed! - Added as 'Speeed' event
 * The pack - Added
 * Inverted Controls - Added as 'Directional confusion' event
 * Porl - Added as 'A piece of history' event, spawns a random lore pearl
 * I’m feeling lucky - Added as 'Secret event'
